schema/models_directory/configuration_interaction/modifiers.json

```diff
@@ -0,0 +1,13 @@
+{
+    "allOf": [
+        {
+            "$ref": "../../model/modifiers.json"
+        }
+    ],
+    "description": "Modifiers specific to this model type only.",
+    "items": {
+        "enum": [
+            "ci-specific-modifier"
+        ]
+    }
+}
```

Comment:
This was a work-in-progress file to illustrate the point - we can clean it up now